spec/inheritance_spec.cr

```diff
@@ -90,4 +107,86 @@ describe "Classes inheriting GObject::Object" do
     LibGObject.g_type_check_instance_is_a(obj, UserSubject.g_type).should eq(1)
     obj.string.should eq("hey")
   end
+
+  it "can set GObject properties" do
```

Comment:
Better to create a `properties_spec.cr` file with the properties spec instead of put the tests in inheritance specs.

Reply:
This test is testing if you can get the value of Crystal declared GObject properties, i.e. you set them _"in C"_ and get them in Crystal, so I think the test case names are swapped.

Reply:
In "can set GObject properties" we check if we correctly implemented `unsafe_do_set_property` and in `can get GObject properties` we check if we correctly implemented `unsafe_do_get_property`. I think swapping it would be confusing.

Reply:
> Better to create a properties_spec.cr file with the properties spec instead of put the tests in inheritance specs.
There is already a `properties_spec.cr` and I don't know if this really fits there.